pkgs/by-name/sh/shogihome/package.nix

```diff
@@ -0,0 +1,136 @@
+{
+  lib,
+  stdenv,
+  buildNpmPackage,
+  fetchFromGitHub,
+  makeWrapper,
```

Comment:
Prefer `makeBinaryWrapper`

Reply:
Can we use `makeBinaryWrapper` even if using dynamic config such as depend of `NIXOS_OZONE_WL`?

https://github.com/NixOS/nixpkgs/blob/48958d802b911408c6432d6aba9fd13c149bf137/pkgs/applications/misc/1password-gui/linux.nix#L139-L155

https://github.com/NixOS/nixpkgs/issues/172583